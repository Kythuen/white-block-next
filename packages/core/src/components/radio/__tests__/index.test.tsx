import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Radio from '../radio.vue'
// import { useRadio } from '../use'

describe('# Radio', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount({
        render() {
          return <Radio />
        }
      })
      // expect(wrapper.classes()).toEqual(['wb-radio'])
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  // describe('## Slots', () => {})

  // describe('## Events', () => {})

  // describe('## Props', () => {})

  // describe('## Composables', () => {})
})
