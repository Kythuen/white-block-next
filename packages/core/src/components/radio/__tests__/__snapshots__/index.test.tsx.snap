// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# Radio > ## Base > ### default config 1`] = `
<label
  class="wb-radio wb-radio--primary wb-radio--md"
  color="$wb-radio-text"
  cursor="pointer"
  flex="inline"
  gap="$wb-radio-gap"
  h="$wb-radio-height"
  items="center"
  p="x-$wb-radio-padding"
  rounded="$wb-radio-radius"
  select="none"
  tabindex="0"
>
  <input
    cursor="pointer"
    h="$wb-radio-size"
    m="0"
    overflow="hidden"
    tabindex="-1"
    type="radio"
    w="$wb-radio-size"
  />
  <div
    class="wb-radio-content"
    flex=""
    gap="$wb-radio-gap"
    items="center"
    truncate=""
    w="full"
  >
    <!---->
  </div>
</label>
`;
